pkg/commands/start_test.go

```diff
@@ -106,6 +125,10 @@ func dnsLookup(t *testing.T, resolverPort, resolverProtocol, domain string) ([]n
 	return resolver.LookupIP(ctx, "ip4", domain)
 }
 
+func randomPort() string {
+	return fmt.Sprint(rand.Intn(32767) + 32768)
+}
+
```

Comment:
This does not guarantee that the port is not currently in use. Please use the mechanism from https://github.com/lima-vm/lima/blob/3aca16d2a762fdb1ba9f50d6749cb03240d0a963/pkg/hostagent/hostagent.go#L181-L223 to pick a random available port.

Reply:
@jandubois I have a bigger PR coming up where it implements the random ports if user does no specify, can we merge this in the changes you suggested will be there.